dts/bindings/sensor/ti,tmp435.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2024 Bittium Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: Texas Instruments TMP435 temperature sensor
+
+compatible: "ti,tmp435"
+
+bus: tmp435
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  external-channel:
+    description: External temperature measurement is enabled
+    type: boolean
+  resistance-correction:
+    description: Resistance correction feature for the external
+                 temperature channel. If not enabled then conversion
+                 is faster but with lower accuracy.
```

Comment:
Please fix the whitespace formatting to match the convention in other bindings

Reply:
Done